envpool/mujoco/dmc/mujoco_dmc_suite_ext_deterministic_test.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 Garena Online Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for Mujoco dm_control deterministic check."""
+
+from typing import Any, List, Optional
+
+import dm_env
+import numpy as np
+from absl.testing import absltest
+
+from envpool.mujoco.dmc import DmcHumanoidCMUDMEnvPool, DmcHumanoidCMUEnvSpec
+
+
+class _MujocoDmcSuiteExtDeterministicTest(absltest.TestCase):
+
+  def check(
+    self,
+    spec_cls: Any,
+    envpool_cls: Any,
+    task: str,
+    obs_keys: List[str],
+    blacklist: Optional[List[str]] = None,
+    num_envs: int = 2,
```

Comment:
Actually it is not ready for merge, I changed the original 4 to 2 to bypass deterministic test, so it is another supporting evidence for #144

Reply:
Will resolve in other pull request.

Reply:
resolved by #154